src/clib/pio_darray_int.c

```diff
@@ -52,21 +52,15 @@ void bpool_free(void *p)
  */
 int compute_buffer_init(iosystem_desc_t *ios)
 {
+    /* Default block size increment = 32 MB */
+    const bufsize DEFAULT_BUF_INC_SZ = 32 * 1024 * 1024;
```

Comment:
It is better to use 32L * 1024L * 1024L here.
gcc gives a warning on code like below: integer overflow in expression [-Woverflow]
```
long x = 1024 * 1024 * 1024 * 1024;
```

Reply:
ok, fixing and rebasing the branch